tests/gpflow/kernels/test_kernels.py

```diff
@@ -65,6 +65,28 @@ def test_rbf_1d(variance, lengthscale):
     assert_allclose(gram_matrix, reference_gram_matrix)
 
 
+@pytest.mark.parametrize("variance, lengthscale", [[2.3, 1.3]])
+def test_cos_1d(variance, lengthscale):
+    X = rng.randn(3, 1)
+    kernel = gpflow.kernels.Cosine(lengthscale=lengthscale, variance=variance)
+
+    gram_matrix = kernel(X)
+    reference_gram_matrix = variance * np.cos((X - X.T) / lengthscale)
+
+    assert_allclose(gram_matrix, reference_gram_matrix)
+
+
+@pytest.mark.parametrize("variance, lengthscale", [[2.3, 1.4]])
+def test_sinc_1d(variance, lengthscale):
+    X = rng.randn(3, 1)
+    kernel = gpflow.kernels.Sinc(lengthscale=lengthscale, variance=variance)
+
+    gram_matrix = kernel(X)
+    reference_gram_matrix = variance * np.sinc((X - X.T) / np.pi / lengthscale)
```

Comment:
why does this one have a `np.pi` in it whereas the `Sinc` implementation doesn't?

Reply:
There seem to be two alternative definitions for sinc. The one with the $\pi$ is called centered sinc, and is the one implemented in numpy.